core/src/main/java/com/yahoo/oak/Chunk.java

```diff
@@ -762,7 +759,7 @@ DescendingIter descendingIter(ThreadContext ctx, K from, boolean fromInclusive,
     // specifier whether the end boundary check needs to be performed on the current scan output
     enum IterEndBoundCheck {
         NEVER_END_BOUNDRY_CHECK,
```

Comment:
Maybe also here BOUNDARY instead of BOUNDRY?

Reply:
fixed